frontend/components/Domain/Recipe/RecipeOrganizerSelector.vue

```diff
@@ -41,7 +43,7 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, ref, useContext, computed, onMounted } from "@nuxtjs/composition-api";
+import { defineComponent, ref, useContext, computed, onMounted, watch } from "@nuxtjs/composition-api";
```

Comment:
@Kuchenpirat is the inclusion of watch intentional? I'm not seeing anywhere it's used, but wondering if it's some weird nuxt thing..?

Reply:
No, that is just an artefact i forgot to remove. 👍
Updated the branch